tests/interfaces-home/task.yaml

```diff
@@ -0,0 +1,144 @@
+summary: Ensure that the home interface works.
+
+details: |
+  The home interface allows a snap to access non-hidden files in $HOME
+
+  A snap which defines the home plug must be shown in the interfaces list.
+  The plug must be autoconnected on install and, as usual, must be able to be
+  reconnected.
+
+environment:
+  SNAP_NAME: home-consumer
+  SNAP_FILE: "./$[SNAP_NAME]_1.0_all.snap"
+  READABLE_FILE: "$HOME/readable"
+  HIDDEN_READABLE_FILE: "$HOME/.readable"
+  WRITABLE_FILE: "$HOME/writable"
+  PREVIOUS_CONTENT: "previous content"
```

Comment:
Can we please use `previous` simply? No need for variables obscuring the logic. This doesn't even save us from typos since the environment var name is also dynamically scoped.